README.md

```
@@ -1,2 +1,71 @@
-# 37-APPJAM-SERVER-SMASHING
-DIVE SOPT 37기 앱잼 스매싱 서버 repository 💪
\ No newline at end of file
+<img width="120" src="https://github.com/user-attachments/assets/e0c8861b-77f5-4878-85ad-0e6308fa2814" />
+
+# SMASHING
+
+<p align="center">
+<img width="5760" height="3240"  src="https://github.com/user-attachments/assets/56258869-67c6-4a17-adc5-9fc4c4946536" />
```

markdownlint-cli2 (0.18.1)

Images should have alternate text (alt text)

(MD045, no-alt-text)